### ISSUE TITLE
[FEATURE] RK auto-roll for Investigate exploration activity

### ISSUE THREAD

**Issue by weepingminotaur** [reyzor1991/pf2e-wiki#10]
The Investigate exploration activity should auto-roll a Recall Knowledge check (similar to what happens with Search or Avoid Notice). Would be super if it could trigger either Workbench's BAM RK or Symon's RK macro.

**Comment by reyzor1991** [comment]
It's hard to use Workbench macro here because, not very clear what number need to store
<img width="284" alt="Screenshot 2023-11-30 at 00 28 28" src="https://github.com/reyzor1991/pf2e-wiki/assets/6688366/dd04d5f0-d2d5-439f-8ddf-fec5546479c9">

**Comment by weepingminotaur** [comment]
I don't think you need to store a number. The GM can check against the different skills to see if the RK yields any info. As long as the activity prompts the RK check, and there's the same option to roll again through the effect icon, it's useful.
